content/post/2018-06-05_badcomkillsgoodanalytics.md

```diff
@@ -0,0 +1,88 @@
+---
+author: Randy Clinton
+title: "Bad communication kills good analytics"
+cover: MISSING
+publishDate: UNKNOWN
+tags:
+ - teams
+ - communication
+ - presentation
+---
+
+Poor communication within an Analytics team and between that team and the company, leave highly skilled analytics employees solving the wrong questions, lacking support for big ideas and leaving the company unsatisfied with their work. Here, I'll discuss ways a team can improve by focusing on constructive conversions.
+<!--more-->
+
+Analytics team exist because companies wants better conversations. Technical tools allow us ways to find information, but at the end we need to talk about it. Without an analytics team, a company will discuss business decisions like marketing spending, product offerings, and ways to encourage customers repurchase. A productive analytics team grounds these conversations in truth and testing.
+
+Four constructive conversations distinguish high functioning analytics teams:
+
+#### Constructive Conversations
+* Talk as a team
+* Talk about your work
+* Talk about the business
+* Talk about where the data is buried
+
+## Talk as a team
+Each week your company has a number of people digging into data, reading documentation for various programming language modules, and attempting to solve business questions.
+
+Teams focusing on talking together spend less time re-learning the same thing. Their members spend less time being frustrated in solitude, and they enjoy their work because they’re learning more.
+
+#### Improving “talk as a team”
```

Comment:
I'd strike this header and just append a sentence to the preceding paragraph like "Here are some ways I've found to promote good intra-team communication."

Reply:
ok